dev_apps/first-actor/buildsign.md

```diff
@@ -0,0 +1,52 @@
+# Signing your actor
+
+First off, lets build and sign the actor as-is.  This way, we can see if the module is behaving as we expect.
+
+
+<details>
+  <summary>Rust</summary>
+  Build it!
+  `cargo build --release`{{execute}}
+  Sign it!
+  `wash claims sign target/wasm32-unknown-unknown/release/calculator.wasm -c wasmcloud:httpserver --name "calculator" --ver 0.1.0 --rev 0`  
```

Comment:
the wasmcloud:httpserver is a first-class (well known) claim name. We should avoid using `-c` for anything other than custom (non-first-party) claims. The proper switch for `wasmcloud:httpserver` is `-q`

Reply:
I would suggest using `--http_server` for clarity in the intro tutorial